src/types/tracker.ts

```diff
@@ -67,22 +106,28 @@ export type TrackedEnvironmentData = {
   browser_version: string
 }
 
+export type AnalyticsEventProperties = {
```

Comment:
this is the old analytics model right?

Reply:
These are some of the tracked properties. I don't have the full list of tracked properties, Gokhan is working on listing them all for all platforms. It's outside of the scope of this story